include/blocksci/chain/transaction.hpp

```diff
@@ -23,13 +23,18 @@
 #include <chrono>
 
 namespace blocksci {
-    class uint256;
+    class uint256;chain
```

Comment:
This typo breaks compilation

Reply:
Oh, I did not see that and haven't tested compilation as I thought I just edited commented content, sorry. Fixed in b86b0a0.